environment.yml

```diff
@@ -12,6 +12,7 @@ dependencies:
 - matplotlib
 - pip:
   - mne>=0.19.1
+  - mayavi
```

Comment:
please no! We are keeping dependencies of mne-bids as light as possible

Reply:
Can I flag circle to skip it in the tests? Or should I put this whole example on mne-python? Or other options too?

Reply:
just make a separate install in circle if it's really needed otherwise I'd say just remove the dependency

Reply:
It's needed for the fiducial alignment GUI. It could just be a comment that says run the GUI instead of actually doing it maybe?

Reply:
yes I prefer that

Reply:
What do you think keep and separate install or comment instead?